index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="pt-br">
```
O que você acha de colocar em português a linguagem do seu html ? 👯